ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/BackingStore/InMemory.hs

```diff
@@ -109,9 +110,15 @@ newTVarBackingStoreInitialiser tracer lookup_ rangeRead_ forwardValues_ count_ e
     pure BackingStore {
         bsClose    = do
             traceWith tracer TVarTraceClosing
```

Comment:
are we interested in having separate trace messages for attempting to close an already-closed backing store?

Reply:
Good point, we'd probably want to make a distinction there